packages/lodestar/src/api/impl/validator/duties/attester.ts

```diff
@@ -0,0 +1,27 @@
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+import {IBeaconDb} from "../../../../db/api";
+import {BLSPubkey, Epoch, ValidatorDuty} from "@chainsafe/eth2.0-types";
+import {assembleValidatorDuty} from "../../../../chain/factory/duties";
+
+export async function getAttesterDuties(
+  config: IBeaconConfig,
+  db: IBeaconDb,
+  epoch: Epoch,
+  publicKeys: BLSPubkey[]
+): Promise<ValidatorDuty[]> {
+  const state = await db.state.getLatest();
```

Comment:
Should we be using the head state?

Reply:
I don't think it matters since attester duties should be stable for current and next epoch

Reply:
I think there might be a case where it would be an issue
```
justified ----- > epoch a -------> epoch b ------> epoch c (current head here)
             \---> latest state here
```

Reply:
That would mean that we are out of sync?
I think in that case we need to return http 503 - Node not synced